### ISSUE TITLE
Prod theme may contain files removed from dev theme

### ISSUE THREAD

**Issue by benoitchantre** [wprig/wprig#591]
## Issue Overview

When there's an existing prod theme before we we run `npm run bundle`, the prod theme may contain files that have been removed from the dev theme.

## Steps to Reproduce

1. Run `npm run bundle`
1. Remove any file from the dev theme
1. Run `npm run bundle` again
1. See that the deleted file remains in the prod theme

## Expected Behavior

Files removed from the dev theme should not be part of the prod theme.

## Possible Solution

Add a first task to empty or remove the prod directory.

Cc @ataylorme

**Comment by ataylorme** [comment]
@benoitchantre I will have to look into this more [but it should already be happening](https://github.com/wprig/wprig/blob/master/gulp/utils.js#L110:L113)

**Comment by ataylorme** [comment]
@benoitchantre I tested this by calling `npm run bundle` to create the production theme. Next, I modified `gulp/utils.js` with:

```js
if ( fs.existsSync( prodThemePath ) ) {
	log( colors.red( `Deleting the existing production theme directory ${ prodThemePath }` ) );
	// and remove it
	rimraf.sync( prodThemePath );
	console.log( fs.existsSync( prodThemePath ) );
	process.exit(1);
}
```

in order to kill the bundle task after the production theme is deleted. As expected, the production theme was removed from my file system.

**Comment by benoitchantre** [comment]
@ataylorme Sorry, I was looking at an incorrect directory because I didn't noticed that my `config.local.json` has been modified. When we run `npm run test:gulp:prod:setup`, `config.local.json` gets modified.

Closed in favor of a #596
